integration-tests/utils/integration-utils.js

```diff
@@ -89,6 +89,17 @@ const findText = (page, textContains, options) => {
   return page.waitForXPath(`//*[contains(normalize-space(.),"${textContains}")]`, options)
 }
 
+const assertTextNotFound = async (page, text) => {
```

Comment:
It would be good to allow the specification of the timeout interval so that the test completes faster. Right now it is using the default timeout period, which slows down the tests a lot when looking for text to not be present.

Reply:
Please do look into shortening the timeout period so the test doesn't take as long.

Reply:
Addressed in 996157c6c37c540cd47781a143a90f61cc2992b5